_includes/manuals/nightly/3.4_install_from_source.md

```diff
@@ -4,9 +4,9 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
 
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
-* Ruby 2.5 or newer
-* NodeJS 10 or newer
-* NPM 3.0 or newer
+* Ruby 2.7 or newer
+* NodeJS 12 or newer
```

Comment:
Or newer is not exactly correct here, we currently do not support Node 16+.

Reply:
Thank you @evgeni, I will fix it.